packages/block-library/src/post-content/block.json

```diff
@@ -1,4 +1,8 @@
 {
 	"name": "core/post-content",
-	"category": "layout"
+	"category": "layout",
+	"context": {
+		"postType": "core/post",
+		"postId": "core/post"
```

Comment:
Again, I don't think we should target blocks explicitly here instead of should have "context names". What happens if I want to build an alternative post block with an alternative UI to select posts.

Reply:
You filter the children you want to support.